### PR TITLE
fixtures: cleanup + safeguard from running in prod;

### PR DESCRIPTION
**Pull request by josketres**

### CODE REVIEW

File: pkg/fixtures/loader.go [hunk 1]
```diff
@@ -6,48 +8,46 @@ import (
...
-	logger = logger.WithChannel("fixture_loader")
+func (f *fixtureLoader) Load(fixtureSets []*FixtureSet) error {
+	settings := fixtureLoaderSettings{}
+	f.config.UnmarshalKey("fixtures", &settings)
 
-	if !config.IsSet("fixture_loader_enabled") {
-		logger.Info("fixture loader is not configured")
-		return nil
-	}
+	f.logger = f.logger.WithChannel("fixture_loader")
```
Comment (applike-ss) [inline pkg/fixtures/loader.go]: This belongs in the NewFixtureLoader func, right?

File: test/test_configs/config.fixtures_dynamodb.test.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
-fixture_loader_enabled: true
-
 aws_cloudwatch_endpoint: http://172.17.0.1:4582
 aws_dynamoDb_endpoint: http://172.17.0.1:4569
 aws_dynamoDb_autoCreate: true
 aws_sdk_retries: 30
+
+fixtures:
```
Comment (applike-ss) [inline test/test_configs/config.fixtures_dynamodb.test.yml]: should we name this fixture_loader instead?

File: pkg/fixtures/loader.go (hunk 1 above)
Comment (josketres) [reply pkg/fixtures/loader.go]: ✅

File: test/test_configs/config.fixtures_dynamodb.test.yml (hunk 2 above)
Comment (josketres) [reply test/test_configs/config.fixtures_dynamodb.test.yml]: Will keep it as it is to be consistent and use the package name.

**Pull request merged**